docs/data/date-pickers/fields/SingleDateFieldExamples.tsx

```diff
@@ -0,0 +1,51 @@
+import * as React from 'react';
+import dayjs from 'dayjs';
+import Typography from '@mui/material/Typography';
+import Box from '@mui/material/Box';
+import Grid from '@mui/material/Grid';
+import { LocalizationProvider } from '@mui/x-date-pickers/LocalizationProvider';
+import { AdapterDayjs } from '@mui/x-date-pickers/AdapterDayjs';
+import { Unstable_DateField as DateField } from '@mui/x-date-pickers/DateField';
+import { Unstable_TimeField as TimeField } from '@mui/x-date-pickers/TimeField';
+import { Unstable_DateTimeField as DateTimeField } from '@mui/x-date-pickers/DateTimeField';
+
+const GridItem = ({
+  label,
+  children,
+  spacing = 1,
+}: {
+  label: string;
+  children: React.ReactNode;
+  spacing?: number;
+}) => {
+  return (
+    <Grid xs={12} item>
+      <Box sx={{ display: 'flex', flexDirection: 'column' }}>
+        <Typography variant="body2" sx={{ mb: spacing }}>
+          {label}
+        </Typography>
+        {children}
+      </Box>
+    </Grid>
+  );
+};
+
+const date1 = dayjs('2022-04-07T14:30:22');
+
+export default function SingleDateFieldExamples() {
+  return (
+    <LocalizationProvider dateAdapter={AdapterDayjs}>
+      <Grid container spacing={4} width={550}>
+        <GridItem label="DateField">
```

Comment:
I aligned this demo with the `DateRangeFieldExamples` just below.
Do you think we should only use this external label system when one of the components in the demo has 2 inputs ?

Reply:
IMHO, I'd say we should go with the style we are going to use eventually.
For DateField components it's possible already—so, why not? 😃 
For DateRangeFields—yeah, we could temporarily stick with label-on-top.

Reply:
Oh, sorry, I missed the fact that we applied similar styling to Validation page as well.
It might have slipped through my eyes, but was there any specific reason to use labels on top of input elements instead of `label` attribute on the picker components like in most other pages?

This is the difference I'm talking about and the bigger portion of our doc uses the latter (like disabled and readonly) style.
![Screenshot 2022-10-12 at 09 23 30](https://user-images.githubusercontent.com/4941090/195265825-6db5831c-b473-4ede-8429-9d08ceae89bf.png)